frontend/src/__tests__/cypress/cypress/e2e/projects/Permissions.cy.ts

```diff
@@ -0,0 +1,315 @@
+import {
+  mockDashboardConfig,
+  mockDscStatus,
+  mockK8sResourceList,
+  mockProjectK8sResource,
+  mockStatus,
+} from '~/__mocks__';
+import { mockRoleBindingK8sResource } from '~/__mocks__/mockRoleBindingK8sResource';
+import { permissions } from '~/__tests__/cypress/cypress/pages/permissions';
+import { be } from '~/__tests__/cypress/cypress/utils/should';
+import { RoleBindingSubject } from '~/types';
+
+const userSubjects: RoleBindingSubject[] = [
+  {
+    kind: 'User',
+    apiGroup: 'rbac.authorization.k8s.io',
+    name: 'user-1',
+  },
+];
+
+const groupSubjects: RoleBindingSubject[] = [
+  {
+    kind: 'Group',
+    apiGroup: 'rbac.authorization.k8s.io',
+    name: 'group-1',
+  },
+];
+
+type HandlersProps = {
+  isEmpty?: boolean;
+};
+
+const initIntercepts = ({ isEmpty = false }: HandlersProps) => {
+  cy.intercept('/api/status', mockStatus());
+  cy.intercept('/api/config', mockDashboardConfig({}));
+  cy.intercept('/api/dsc/status', mockDscStatus({}));
+  cy.intercept(
+    { pathname: '/api/k8s/apis/project.openshift.io/v1/projects' },
+    mockK8sResourceList([mockProjectK8sResource({ k8sName: 'test-project' })]),
+  );
+  cy.intercept(
+    {
+      method: 'GET',
+      pathname: '/api/k8s/apis/rbac.authorization.k8s.io/v1/namespaces/test-project/rolebindings',
+    },
+    {
+      statusCode: 200,
+      body: mockK8sResourceList(
+        isEmpty
+          ? []
+          : [
+              mockRoleBindingK8sResource({
+                name: 'user-1',
+                subjects: userSubjects,
+                roleRefName: 'edit',
+              }),
+              mockRoleBindingK8sResource({
+                name: 'group-1',
+                subjects: groupSubjects,
+                roleRefName: 'edit',
+              }),
+            ],
+      ),
+    },
+  );
+};
+
+describe('Permissions tab', () => {
+  const userTable = permissions.getTable('project-sharing-table User');
+  const groupTable = permissions.getTable('project-sharing-table Group');
```

Comment:
These's are tables static. Add functions `getUserTable` and `getGroupTable` to the page object.